examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -443,17 +467,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n",
+      "Operator `initdamp` run in 0.01 s\n",
+      "Operator `padfunc` run in 0.01 s\n"
+     ]
+    },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "      fun: 211.63605994835484\n",
+      "      fun: 203.91187255306352\n",
```

Comment:
Is the output expected to change?

Reply:
I guess it is the last value of the objective function, so in principle, it should be the same. I assumed that the result was correct looking at the figures. The final models in both versions of the notebooks are virtually the same and I see absolutely no reason to get different values. Maybe @mloubout can help us understand a little more.

Reply:
I ran several times the old notebook and got the same value as the new 203.91187255306352. I don't know where came from the other value. I confirmed the results are the same for both cases (old and new notebooks) OK.

Reply:
Ok this might have changed because of some previous change and the notebook wasn't rerun